tools/autograd/derivatives.yaml

```diff
@@ -1289,10 +1289,16 @@
   self: "GradMode::is_enabled() ? infinitely_differentiable_silu_backward(grad, self) : silu_backward(grad, self)"
 
 - name: elu(Tensor self, Scalar alpha=1, Scalar scale=1, Scalar input_scale=1) -> Tensor
-  self: elu_backward(grad, alpha, scale, input_scale, result)
+  self: elu_backward(grad, alpha, scale, input_scale, false, result)
```

Comment:
nit, you can add a comment before the hardcoded bools here like `/* is_result */` to make it clear to the reader what it is.

Reply:
Sure! Thanks